components/resources/library/src/jsMain/kotlin/org/jetbrains/compose/resources/ResourceEnvironment.js.kt

```diff
@@ -0,0 +1,22 @@
+package org.jetbrains.compose.resources
+
+import kotlinx.browser.window
+
+private external class Intl {
+    class Locale(locale: String) {
+        val language: String
+        val region: String
+    }
+}
+
+internal actual fun getResourceEnvironment(): ResourceEnvironment {
+    val locale = Intl.Locale(window.navigator.language)
+    val isDarkTheme = window.matchMedia("(prefers-color-scheme: dark)").matches
+    val dpi: Int = (window.devicePixelRatio * 96).toInt()
```

Comment:
Let's make `96` a constant, not just magic number. Or extension function for conversion

Reply:
I will add just a comment here. Constant with the only usage is overkill